code/game/gamemodes/changeling/modularchangling.dm

```diff
@@ -315,6 +315,13 @@ var/list/datum/power/changeling/powerinstances = list()
 	genomecost = 0
 	verbpath = /mob/proc/changeling_toggle_darksight
 
+/datum/power/changeling/changeling_infect
+	name = "Second life"
+	desc = "We infect the victim, leaving a copy of ourselves and our knowledge in case we die."
```

Comment:
We infect a victim with Psychocyst that posses a copy of our knowledge and abilities. It will germinate and take over it's host if we die.